lib/internal/process/per_thread.js

```diff
@@ -258,6 +260,15 @@ function wrapProcessMethods(binding) {
     };
   }
 
+  function loadEnvFile(path) {
```

Comment:
nit: we can mark the arg as optional (so `process.loadEnvFile.length` returns 0)
```suggestion
  function loadEnvFile(path = undefined) {
```